guides/hack/15-unsupported/06-others.md

```diff
@@ -42,4 +42,5 @@ The typechecker doesn't understand aliasing trait methods. It actually believes
 -  Case-insensitive function calls and class lookups
 -  Mixing method call syntax (e.g., using `self` to call a non-static method)
 -  Hack supports [argument unpacking](https://wiki.php.net/rfc/argument_unpacking), except for on arbitrary traversables.
-
+-  The [superglobals](http://php.net/manual/en/language.variables.superglobals.php). They're treated as un-typed variables in partial mode and undefined in strict.
```

Comment:
I do have this section on SuperGlobals

https://github.com/hhvm/user-documentation/blob/master/guides/hack/01-types/08-advanced-rules.md#superglobals

Are we simpatico on my wording and your wording?